src/main/java/io/nats/client/StreamState.java

```diff
@@ -0,0 +1,103 @@
+package io.nats.client;
```

Comment:
License/Copyright

Reply:
Is there a corresponding change to rename StreamInfo to StreamState in the go client?  We'll want to keep the APIs as consistent as we can.

Reply:
fixed